src/Microsoft.Identity.Web/ClientInfo.cs

```diff
@@ -7,37 +7,37 @@
 
 namespace Microsoft.Identity.Web
 {
-    internal class ClientInfo
-    {
-        [JsonPropertyName(ClaimConstants.UniqueObjectIdentifier)]
-        public string UniqueObjectIdentifier { get; set; } = null!;
-
-        [JsonPropertyName(ClaimConstants.UniqueTenantIdentifier)]
-        public string UniqueTenantIdentifier { get; set; } = null!;
-
-        public static ClientInfo? CreateFromJson(string clientInfo)
-        {
-            if (string.IsNullOrEmpty(clientInfo))
-            {
-                throw new ArgumentNullException(nameof(clientInfo), IDWebErrorMessage.ClientInfoReturnedFromServerIsNull);
-            }
-
-            return DeserializeFromJson(Base64UrlHelpers.DecodeToBytes(clientInfo));
-        }
-
-        internal static ClientInfo? DeserializeFromJson(byte[] jsonByteArray)
-        {
-            if (jsonByteArray == null || jsonByteArray.Length == 0)
-            {
-                return default;
-            }
-
-            var options = new JsonSerializerOptions
-            {
-                PropertyNameCaseInsensitive = true,
-            };
-
-            return JsonSerializer.Deserialize<ClientInfo>(jsonByteArray, options);
-        }
-    }
+	internal class ClientInfo
+	{
+		[JsonPropertyName(ClaimConstants.UniqueObjectIdentifier)]
+		public string UniqueObjectIdentifier { get; set; } = null!;
```

Comment:
setting to null! seems like we are incorrectly trying to override the type. The reality is that json can omit the field which would result in a null string.